docs/_legacy/import.md

```diff
@@ -15,7 +15,7 @@ If you have existing integrations built with Zapier's legacy web builder, for no
 
 Once your Zapier Legacy Web Builder integration is converted to Zapier Platform UI, it will look nearly the same as a new integration built in Zapier Platform UI's visual builder. It will include every feature in new Zapier integrations, along with a few extra features to manage parts of your integration that were different in the Legacy Web Builder.
 
-Many things in Zapier Platform UI are similar to the tools you've used in Zapier's Legacy Web Builder, only with a more modern design. You can build full integrations from our form-based editor, often without touching code. If you do need to use code for a custom API call, fetch custom fields, or to parse non-standard response data, you can add JavaScript code for each
+Many things in Zapier Platform UI are similar to the tools you've used in Zapier's Legacy Web Builder, only with a more modern design. You can build full integrations from our form-based editor, often without touching code. If you do need to use code for a custom API call, fetch custom fields, or to parse non-standard response data, you can add JavaScript code for each trigger or action in your app.
```

Comment:
I was originally going to nit about how Trigger and Action are supposed to be capitalized but it looks like the rest of this doc doesn't heavily enforce that so it's safe to let it slide. 😁 I also vaguely recall (maybe in a dream 😂) that we are relaxing that anyways so it's a 👍 from me!

Reply:
At least it's a complete sentence now! :)
I've noticed that there's not always consistency around that, but it looks like the current [style guide](https://brand.zapier.com/content/writing-about-zapier.html) leans toward lowercase.

Reply:
LOL I am a dinosaur: https://zapier.slack.com/archives/CBSVAKJHY/p1570054269001000

> October 2nd, 2019

Reply:
Aw, I was -1 Y 3D old in Zapier years when that happened 🐤